myhw-week1/db.js

```diff
@@ -0,0 +1,103 @@
+const mysql = require('mysql');
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'world'
+});
+
+
+connection.connect();
+
+// Answer number (1)
+let counties_les = "select * from countries where countries.country_population < 80000000"
+connection.query(counties_les, function (error, results) {
+  if (error) {
+    throw error;
+  }
+  console.log(results);
+});
+
+// Answer number (2)
+let countries_land = "select * from countries where country_name like '%land%'"
+connection.query(countries_land, function (error, results) {
+  if (error) {
+    throw error;
+  }
+  console.log(results);
+});
+
+// Answer number (3)
+let cities_pop = "select * from cities where city_population between 500000 and 1000000"
+connection.query(cities_pop, function (error, results) {
+  if (error) {
+    throw error;
+  }
+  console.log(results);
+});
+
+// Answer number (4)
+let europe_countries = "select * from countries where continent = 'europe'"
+connection.query(europe_countries, function (error, results) {
+  if (error) {
+    throw error;
+  }
+  console.log(results);
+});
+
+// Answer number (5)
+let all_countries = "select * from countries order by surface_area desc"
+connection.query(all_countries, function (error, results) {
+  if (error) {
+    throw error;
+  }
+  console.log(results);
+});
+
+// Answer number (6)
+let netherlands_cities = "select countries.country_name, cities.city_name from cities inner join countries on cities.country = countries.country_name where countries.country_name = 'netherlands'"
```

Comment:
please explain the use of inner join.

Reply:
I wanted to join two tables 
first i did specify which columns i want to display by using "select". 
second I used "from" to specify the tables i wanted to join. 
third I did specify which columns I wanted to much by using "on"  and the condition by using "where"

Reply:
ooh I had to use join to get only the matched data.

Reply:
Good explanation! I like that you know what you are doing exactly.